### ISSUE TITLE
Type in instructions for Info.plist on React Native page

### ISSUE THREAD

**Issue by LectricAvenue** [BranchMetrics/react-native-branch-deep-linking-attribution#702]
https://help.branch.io/developers-hub/docs/react-native#section-configure-branch.

Section "Configure App" under iOS and the bullet point Configure Info.plist has the instructions:

Add `branch_universal_link_domains` with your live key domain
Add `branch_universal_link_domains` with your current Branch key

From here: https://help.branch.io/developers-hub/docs/ios-basic-integration#4-configure-infoplist
looks like the second one should be `branch_key`.

Tripped me up for a second so hopefully, this is helpful.

**Comment by jf-branch** [comment]
That is correct @LectricAvenue Thank you for calling that out, i will update the docs now.
